docs/application/dotnet/guides/nui/view.md

```diff
@@ -1,75 +1,133 @@
+<table style="width:100%">
+<tr>
+<td style="width:100%" align="center">
+<img src="./media/view_Figure_ParentOrigin_PivotPoint.png">
+</td>
+</tr>
+</table>
 
-```
-View view = new View();
-view.Size2D = new Size2D(200, 200);
-view.BackgroundColor = Color.Red;
-```
+The following figure shows three cases of parent `View` and its child `View`. Parent `View` is blue and child `View` is red. In the following examples, all of the red `View` is identically located at (3, 5) of its parent coordinate system that is based on `ParentOrigin` property of the child `View`.
 
-**Figure: View with a red background**
+**Figure: The final position with different ParentOrigin**
 
-![View with a red background](./media/background_control_color.png)
+<table style="width:100%">
+<tr>
+<td style="width:100%" align="center">
+<img src="./media/view_Figure_Position.png" width="100%">
+</td>
+</tr>
+</table>
 
-You can handle all existing views similarly. For example, to set the background color for a TextLabel:
+Alternatively, you can use an arbitrary `Position` type value defined in the unit coordinates to set `ParentOrigin` without predefined values.
+
+`Orientation` of `View` is the rotation from its default orientation and `Scale` of `View` is the size ratio between the size to be rendered and the default size. For these two transforms, `View` is rotated and scaled around [PivotPoint](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_PivotPoint). NUI also provides predefined presets of `PivotPoint` identical to `ParentOrigin`. Alternatively, you can use custom values.
+
+`Orientation` and `Scale` of `View` are also affected by the transforms of its parent. In the final calculation of `View`, `Orientation` and `Scale` of parents are first applied to the child `View` with parent's `PivotPoint`, and then `Orientation` and `Scale` of child are applied.
+
+The following examples demonstrate the rotation and scaling that change `Orientation` and `Scale` respectively. **View A** is a child of **View B**. Therefore, transforms of **A** only affects **A**. however, the transforms of parent **B** influences its child **A**. The black dot in the figure is `PivotPoint` that is used by the transform.
+
+**Figure: Rotation and Scaling.**
+
+<table style="width:100%">
+<tr>
+<td style="width:50%" align="center">
+<img src="./media/view_Figure_Orientation.gif" width="100%">
+</td>
+<td style="width:50%" align="center">
+<img src="./media/view_Figure_Scale.gif" width="100%">
+</td>
+</tr>
+</table>
 
-```
-TextLabel label = new TextLabel( "Hello World" );
-label.BackgroundColor = Color.Red;
-```
 
-**Figure: TextLabel object with a red background**
+<a name="2"></a>
+## Directional Navigation
+
+NUI also provides directional navigation between `View`s using arrow keys on the keyboard. You can simply set the following properties to specify the next `FocusableView`s for each direction:
```

Comment:
It is recommended to avoid plurals.
For more information, see https://developers.google.com/style/plurals-parentheses

Suggestion:
Can we generalize and write:
"NUI also provides directional navigation between Views using arrow keys on the keyboard. You can simply set the following properties to specify the next `FocusableView` for each direction:"

Please verify and let me know your opinion on the same.

Please consider the same for lines 80 and 101.

Reply:
I agree with you.
But how about to use "between each `View`". Only one View without mark is little weird...

And also...
in the line number 80, I couldn't find prefer way to switch 'events' to `Event`. I think just "variety of events" and "basic events" are the best way.

And, in the line number 101, I simply removed "s" of first `Event`, and use "is" instead of "are". Is it acceptable?